lambdas/ndjson-builder/README.md

```diff
@@ -0,0 +1,10 @@
+## ndJSON builder function
+
+Handler for a lambda which receieves messages from a SQSEventSource trigger containing a list of STAC JSON files. Fetch and build the data into ndJSON files, upload to 
```

Comment:
```suggestion
Code intended to receive messages from an SQSEventSource trigger. Those messages contain a list of STAC JSON file URLs. Fetches and build the data into ndJSON files, upload to 
```

I think the messages received by the ndjson builder lambda include URLs to the STAC JSON files right? I think saying URLs makes this more explicit.